cluster/examples/kubernetes/ceph/cluster.yaml

```
@@ -222,16 +222,40 @@ spec:
 # nodes below will be used as storage resources.  Each node's 'name' field should match their 'kubernetes.io/hostname' label.
     # nodes:
     #   - name: "172.17.4.201"
+    #     resourceClassName: "class1"
     #     devices: # specific devices to use for storage can be specified for each node
     #       - name: "sdb"
     #       - name: "nvme01" # multiple osds can be created on high performance devices
+    #       - nameL "hdd"
     #         config:
     #           osdsPerDevice: "5"
+    #         resourceClassName: "class2"
     #       - name: "/dev/disk/by-id/ata-ST4000DM004-XXXX" # devices can be specified using full udev paths
     #     config: # configuration can be specified at the node level which overrides the cluster level config
     #   - name: "172.17.4.301"
     #     deviceFilter: "^sd."
+    #     resourceClassName: "class2"
+    #     devices:
+    #       - name: "vdc"
+    #       - name: "hdd"
+    #         resourceClassName: "class1"
+        # - name: "minikube-m02"
+        #   resources: # default for all OSDs in this node
+        #     limits:
+        #       cpu: "250m"
+        #       memory: "128Mi"
     # when onlyApplyOSDPlacement is false, will merge both placement.All() and placement.osd
+    # resourceClasses:
+    #   - name: class1
+    #     resources: # default for all OSDs in this node
```

no need for this comment here
```suggestion
    #     resources:
```